### PR TITLE
Update mars.txt

### PR DESCRIPTION
**Pull request by martaciechonska**
Add an extra line in your file.

This is the extra message.

### CODE REVIEW

**Comment by serenading** [comment]
cool, I'll have a look

File: mars.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 cold and dry.
 Two moons may be a bit confusing.
 A lot of red color could be nice.
+I'm changing your file!
```
Comment (serenading) [inline mars.txt]: Maybe 3 exclamation marks work better

Comment (martaciechonska) [inline mars.txt]: OK, will add!!

**Comment by martaciechonska** [comment]
OK, found the conversation!!

**Pull request merged**